### DISCUSSION TITLE
Expose new data to clients extension.

### DISCUSSION THREAD

**Discussion by Tolriq** [opensubsonic/open-subsonic-api#52] [Draft extensions]
See the proposal here:

https://github.com/opensubsonic/open-subsonic-api/pull/51

The PR will have a link to up to date documentation exposing all the new data including multi valued fiels, contributors, ...

@opensubsonic/clients @opensubsonic/servers 

Do not forget to comment and vote here.

**Comment by Tolriq** [comment]
For the record this is the first step to start moving. Exposing new data allows clients to vastly improve without much changes for servers who already have the data available.

The next step after this, will be to see what filters we add where to allow clients to only get the data they want without having to do the filtering on their side.

**Comment by lachlan-00** [comment]
can all the different label fields just be called 'label'? using albumlabel and recordlabel aren't adding extra information

**Reply by Tolriq** [reply]
There only recordlabel now I'll fix the json part. As explained in the PR, Label have many different meanings. In many APIs a label is a title/name.
